docs/docs/03-interactivity-and-dynamic-uis.zh-CN.md

```diff
@@ -11,22 +11,26 @@ next: multiple-components-zh-CN.html
 ## 简单例子
 
 ```javascript
-var LikeButton = React.createClass({
-  getInitialState: function() {
-    return {liked: false};
-  },
-  handleClick: function(event) {
+class LikeButton extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      liked: false
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+  handleClick() {
     this.setState({liked: !this.state.liked});
-  },
-  render: function() {
-    var text = this.state.liked ? 'like' : 'haven\'t liked';
+  }
+  render() {
+    const text = this.state.liked ? 'like' : 'haven\'t liked';
     return (
-      <p onClick={this.handleClick}>
+      <div onClick={this.handleClick}>
         You {text} this. Click to toggle.
-      </p>
+      </div>
     );
   }
-});
+}
```

Comment:
Can you leave this example as is? We're not using ES6 classes broadly yet and this should match the code in the English version.